django/otto/fixtures/apps.yaml

```diff
@@ -6,6 +6,14 @@
     prod_ready: true
     user_group: AI assistant user
     features:
+      - fields:
+          short_name: "chat"
+          name_en: Chat
+          name_fr: Discuter avec l'IA
+          description_en: Chat with a Justice-hosted AI (like ChatGPT).
+          description_fr: Discuter avec une IA hébergée par Justice (comme ChatGPT).
+          url: chat:chat_with_ai
+          category: ai_assistant
```

Comment:
Move Chat to the top of the list in app tiles - and the first tab in the AI assistant.